### PR TITLE
Fix memory leaks

### PR DESCRIPTION
**Pull request by owaddell**
This pull request
* fixes a memory leak in an error path of `sqlite:marshal-bindings`
* fixes a memory leak in `console-event-handler`
* fixes a few places where we need to close ports
* adds notes on running Swish with Address Sanitizer

It also
* adds a missing test for `spawn-os-process-detached`
* adds a `-b` option to `run-mat` to simplify running a specific mat using the mat-prereq artifacts
* reorders some definitions to avoid `letrec*` checks
* quotes paths in `make install` targets
* updates the `.gitattributes` file

### CODE REVIEW

File: src/swish/http.ss [hunk 1]
```diff
@@ -211,14 +211,16 @@
                              (conn:start&link cache-mgr ip op options)])
                            (http:handle-input conn url-handler options)))))))])
           (monitor pid)
-          `#(no-reply ,($state copy* [pid->op (ht:set pid->op pid op)])))]
+          `#(no-reply ,($state copy* [pid->op (ht:set pid->op pid (cons ip op))])))]
```
Comment (burgerrg) [inline src/swish/http.ss]: Should the name `pid->op` change to `pid->ports`?

**Review by burgerrg — APPROVED** [review]
Thanks for getting the address sanitizer to work!

File: src/swish/http.ss (hunk 1 above)
Comment (owaddell) [reply src/swish/http.ss]: Good thought!

**Comment by owaddell** [comment]
Thanks for the quick review, Bob!

**Pull request merged**